Data/Sys/GameSettings/SHU.ini

```diff
@@ -0,0 +1,14 @@
+# SHUE52 - Cabela's Dangerous Hunts 2011: Special Edition
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[OnFrame]
+# Add memory patches to be applied every frame here.
+
+[ActionReplay]
+# Add action replay cheats here.
```

Comment:
Minor nitpick, I recall discussing this when we flipped the EFB default that there's no point to include these sections if nothing is in them.
I don't particularly care if they stay though since other games do this too.

Reply:
Eh, yeah, might as well not add them. It might be worth getting rid of them from everywhere (I removed the completely useless OnLoad sections in #13138).